dependencies/common/install-npm-dependencies

```diff
@@ -85,7 +85,7 @@ if [ ! -x "$(command -v yarn)" ]; then
   rm $YARN_INSTALL_SCRIPT
   INSTALL_PATH="${HOME}/.yarn/bin:${HOME}/.config/yarn/global/node_modules/.bin:${INSTALL_PATH}"
 else
-  YARN_VERSION=`PATH=${INSTALL_PATH} yarn --version`
+  YARN_VERSION="PATH=${INSTALL_PATH} yarn --version"
```

Comment:
This doesn't look correct to me. Do you want to quote `${INSTALL_PATH}` within the call? E.g.

```suggestion
  YARN_VERSION="$(PATH="${INSTALL_PATH}" yarn --version)"
```

You can also use [shellcheck](https://github.com/koalaman/shellcheck) which is pretty good at spotting these.

Reply:
I don't think you want to get rid of the backquotes, how about:

```suggestion
YARN_VERSION=`$(PATH="${INSTALL_PATH}" yarn --version)`
```

Reply:
Oops! Great catch

Reply:
I don't think you want backquotes and `$()`, they do the same thing. It should either be `$(PATH="${INSTALL_PATH}" yarn --version)` or `<backtick>PATH="${INSTALL_PATH}" yarn --version)<backtick>`